nh_food_and_fluid/static/src/js/chart.js

```diff
@@ -143,10 +143,13 @@ var fluidTableDescription = {
 };
 
 function drawFood_fluidTable(settings, serverData){
-    serverData.sort(function (a, b) {
-        var dateA = new Date(a.effective_date_terminated).getTime(), dateB = new Date(b.effective_date_terminated).getTime();
-        return dateA - dateB;
-    });
+    for ( var idx = 0; idx < serverData.length; idx++ ) {
+        var data = serverData[idx];
+        data.observations.sort( function (a, b) {
```

Comment:
Does this not require re-assignation (`data.observations = data.observations.sort(....`)  ?

Reply:
Not this time. sort() changes inline rather than assigns to a new array.